artifacts/definitions/SUSE/Linux/Events/Timers.yaml

```diff
@@ -0,0 +1,38 @@
+name: SUSE.Linux.Events.Timers
+description: |
+   Print events when a systemd timer gets added/deleted/executed
+
+type: CLIENT_EVENT
+
+sources:
+  - name: TimerStateChange
+    precondition: SELECT OS From info() where OS = 'linux'
+    description: Collect event when a new timer is started or stopped
+    query: |
+      SELECT timestamp(string=REALTIME_TIMESTAMP) as Time,
+              JOB_TYPE AS Action,
+              UNIT As Timer
+        FROM watch_journal()
+        WHERE SYSLOG_IDENTIFIER = "systemd" AND CODE_FUNC = "job_emit_done_message" AND UNIT =~ ".*timer"
+
+  - name: TimerExecs
+    precondition: SELECT OS From info() where OS = 'linux'
+    description: Collect systemd timer executions from journal
+    query: |
+      LET timers = SELECT parse_json_array(data=Stdout) AS list
+        FROM execve(argv=['systemctl', 'list-timers', '--all', '-o', 'json', '--no-pager'])
+      
+      LET timer_exec = SELECT REALTIME_TIMESTAMP, _PID, MESSAGE, SYSLOG_IDENTIFIER, _SYSTEMD_UNIT, _UID, {SELECT activates from timers.list} AS activates
+        FROM watch_journal()
+        WHERE _SYSTEMD_UNIT =~ ".*service" AND _SYSTEMD_UNIT IN activates
```

Comment:
I tested with client running under systemd and the cpu went very high. The problem is that the client itself is logging these to the journal which leads to a loop
```
Dec 07 10:46:45 sle15sp5vrr velociraptor-client[15250]: [INFO] 2023-12-07T10:46:45Z shell: Running external command [systemctl list-timers --all -o json --no-pager]
```
I hacked it to comfirm, but maybe there are better ways to fix it
```
WHERE _SYSTEMD_UNIT != "velociraptor-client.service" AND _SYSTEMD_UNIT =~ ".*service" AND _SYSTEMD_UNIT IN activates

```

Reply:
With the new implementation this shouldn't be an issue anymore.